### DISCUSSION TITLE
RosettaDock in Rosie

### DISCUSSION THREAD

**Discussion by ngandhiin** [RosettaCommons/rosetta#72] [ROSIE]
I have performed Docking (protein-protein complex) using RosettaDock-5.0. I couldn't find manual related to the interpretation of the results. In the folder, I can find 30 pdb files each for receptor and ligand. I can also find relax scores for both ligands and complex. The 30 files shows presence of a turn at the N-terminal of ligand which correlates with the literature. 

My question is how to analyse the data and which pairs will form complexes? Is it based on RMSD or scores or something else.

Thank you for your kind consideration.

Neha

**Comment by roccomoretti** [comment]
It depends a bit on the protocol you're using and the exact post-analysis you're looking for, but generally speaking the best docked structure is selected by energy. In the ideal case, the lowest energy structure will be the one which is closest to the actual experimental results.

Unfortunately, this is not always the case. There's pathologies in the Rosetta energy function which means that certain non-native structures score abnormally low. But usually those are very "precarious" structures, where small changes in structure result in much worse scores. With typical native structure, the energy basin they lie in is deep and wide. So when you do docking runs it's often worth clustering structures. Structures from a cluster which is both low-energy and large are more likely to be native-like than something which is low energy, but without a bunch of nearby low energy structures.

A quick check of this (which doesn't require all-against-all clustering) is to look a the score-vs-rmsd plot. If you don't have an actual native by which to calculate the rmsd, you can use a low energy structure as a putative native structure. In the ideal case your score-vs-rmsd plot will be a "funnel", where the low energy structures are all low-energy, and as you increase in energy, they spread out. (Note that the energy function is rugged, so low-rmsd structures can still have bad energies) This doesn't need to be 100%. You can have a few low-energy+high-rmsd structures, so long as they look like outliers. -- If your score-vs-rmsd plot is decently funnel shaped, that gives some indication that the putative native structure is probably close to the actual native structure.

**Comment by roccomoretti** [comment]
Marking this as "closed", as I hope your question has been answered. If that assessment is incorrect, feel free to re-open or open up a new discussion with more details.
